source/_integrations/rfxtrx.markdown

```diff
@@ -344,11 +344,22 @@ action:
     event: 0b1111e003af16aa10000060
 ```
 
+Alternatively:
+
+[![Open your Home Assistant instance and show your service developer tools.](https://my.home-assistant.io/badges/developer_services.svg)](https://my.home-assistant.io/redirect/developer_services/)
```

Comment:
Please don't use the badges, just link inline text.

Additionally, use the `{% my .. %}` plugin used in the documentation instead of hard Markdown/static links.